receiver/sqlserverreceiver/documentation.md

```diff
@@ -260,6 +279,43 @@ This metric is only available when the receiver is configured to directly connec
 | logical_filename | The logical filename of the file being monitored. | Any Str |
 | file_type | The type of file being monitored. | Any Str |
 
+### sqlserver.database.io.write_latency
```

Comment:
How long ago we introduced `sqlserver.database.io.read_latency`? Can we have `sqlserver.database.latency` with `direction`? That would resolve the previous comment

Reply:
It was introduced in `v0.100.0`. @djaglowski @StefanKurek Are you okay with a breaking change to rename `sqlserver.database.io.read_latency` as `sqlserver.database.latency` with a `direction` attribute here?

Reply:
Makes sense to me

Reply:
I've renamed and added a breaking changelog entry. 👍